build.sbt

```diff
@@ -4,7 +4,7 @@ import sbt._
 val PlayVersion = "2.5.4"
 val Scala10 = "2.10.5"
 val Scala11 = "2.11.8"
-val ProjectVersion = "0.2.2"
+val ProjectVersion = "0.2.3"
```

Comment:
@slavaschmidt, no, in Travis: https://travis-ci.org/zalando/api-first-hand/builds/221267940
First, it's
```
[info] Resolving de.zalando#beard_2.10;0.1.2 ...
```

and later, in sbt-scripted:
```
[info] [info] Resolving de.zalando#beard_2.10;0.0.6 ...
```